Makefile

```diff
@@ -6,8 +6,13 @@ VERSION          ?= 0.0.0
 SERVICE_PORT     ?= 3000
 DOCKER_REGISTRY  ?= docker.io/tosone
 DOCKER_PLATFORMS ?= linux/amd64,linux/arm64
-
+APPNAME          ?= sigma
+NAMESPACE        ?= default
```

Comment:
namespace use `sigma`, and `helm install --create-namespace` will create this ns automatically.